### PR TITLE
Feature increases2

### PR DESCRIPTION
**Pull request by atm-aurelien**
NEW: propal creation goes to third-party search if const is defined
NEW: propal contacts tables are separated for a better view

### CODE REVIEW

File: htdocs/comm/propal.php [hunk 1]
```diff
@@ -1217,7 +1218,12 @@
 /*
  * View
  */
-
+ 
+ // If const defined, creation = society search
+if ($action == 'create' &&	dolibarr_get_const($db, 'PROPAL_NEW_SEARCH_SOC')) {
+	header("Location: ../societe/list.php?leftmenu=thirdparties");
+	exit();
```
Comment (rdoursenaud) [inline htdocs/comm/propal.php:1225]: This code is unreachable and useless plus exit should only be used in CLI scripts, not CGI.

**Comment by rdoursenaud** [comment]
I don't understand the purpose of this. Could you explain a bit more?

**Comment by atm-aurelien** [comment]
This code force users to cearch a third-party before creating a commercial proposal.
By this way, this limits third-parties duplicates.

**Comment by rdoursenaud** [comment]
Sorry, I still don't understand the purpose.
If I create a new proposal, I can already search for thirdparties in-place:
capture d ecran de 2015-09-07 18-55-46
And there's no way to create a new third party from there, so no duplication.
Am I missing something obvious?

**Comment by eldy** [comment]
I think goal is to have a process of creating a prospect that start always by the search page on thirdparty providing a way to search not only on name but also on professional id like siren, or zip, or email..

For code, can you prefer to use 
if (! empty($conf->global->PROPAL_NEW_SEARCH_SOC)) 
instead of 
dolibarr_get_const($db, 'PROPAL_NEW_SEARCH_SOC'))

dolibarr_get_const is dedicated for admin page and make database access. This is useless, all vars are already loaded into $conf->global array

**Comment by eldy** [comment]
With last version, we can search on thirdparty from every screen. So i close request

**Pull request closed without merging**